src/Tizen.NUI/src/public/ViewProperty/TextShadow.cs

```diff
@@ -0,0 +1,165 @@
+﻿/*
+ * Copyright(c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System.ComponentModel;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// The Text Shadow for TextLabel.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class TextShadow : Internal.ICloneable
+    {
+        private Color color = Color.Black;
+        private Vector2 offset = Vector2.Zero;
+        private float blurRadius = 0.0f;
+
+        private PropertyMap propertyMap = null;
+
+        internal delegate void PropertyChangedCallback(TextShadow instance);
+        internal PropertyChangedCallback OnPropertyChanged = null;
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public TextShadow()
+        {
```

Comment:
The constructor have to have all 3 arguments to set readonly properties.

Reply:
Done.

Reply:
Done.